antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlLexer.g4

```diff
@@ -1002,7 +1002,7 @@ fragment SQUOTA_STRING
     ;
 
 fragment BQUOTA_STRING
-    : '`' ( '\\'. | '``' | ~('`'| '\\') )* '`'
+    : '`' ( '\\' ~('`') | '``' | ~('`'| '\\') )* '`'
```

Comment:
?

Reply:
identifier like `\`select` is not allowed. ` in quoted identifier can only be written as ``.